docs/kusama-ledger.md

```diff
@@ -0,0 +1,170 @@
+---
+id: kusama-ledger
+title: Using Ledger Devices
+sidebar_label: Ledger Devices
+---
+
+Kusama has a [Ledger][] application that is compatible with the Ledger Nano S and Ledger Nano X
+devices. The Ledger devices are hardware wallets that keep your secret secured on a physical device
+that does not get exposed to your computer or the internet. It is strongly recommended to use a
+hardware wallet if you are managing a significant amount of funds.
+
+The Kusama application allows you to manage Kusama's native asset, the KSM. It supports most of the
+available transaction types of the network (a notable exception is the "Batch" transaction from the
+Utilty pallet).
+
+## Requirements
+
+This is a list of what you will need before starting:
+
+- A Ledger Nano S or a Ledger Nano X.
+- The latest firmware installed. (At the time of writing this was 1.6)
+- Ledger Live is installed.
+- A web browser is installed that you can use to access [Polkadot-JS Apps][].
+
+## Installing the Ledger Application
+
+### Using Ledger Live
+
+> NOTE: If you are reading this notice the Kusama application has not been approved for Ledger Live
+> yet. If you would like to proceed with the pre-release installation please follow the "Using the
+> Developer Release" instruction below. Otherwise hold tight, the application should be approved
+> soon!
+
+- Open the "Manager" tab in Ledger Live.
+- Connect and unlock your Ledger device.
+- If asked, allow the manager on your device by pressing the right button.
+- Find Kusama in the app catalog.
+- Click the "Install" button of the app.
+- An installation window appears.
+- Your device will display "Processing…"
```

Comment:
```suggestion
- Your device will display "Processing…".
```

Reply:
Should there be a period if there is already an ellipsis inside quotes?  I feel like this is an "American vs British English" thing...

Reply:
Huh - good question. I normally do this myself when writing (that's what I grew up learning).